handsontable/src/plugins/copyPaste/copyPaste.js

```diff
@@ -626,13 +626,15 @@ export class CopyPaste extends BasePlugin {
   onCopy(event) {
     const focusedElement = this.hot.getFocusManager().getRefocusElement();
     const isHotInput = event.target?.hasAttribute('data-hot-input');
+    const selected = this.hot.getSelected();
+    const TD = selected ? this.hot.getCell(selected[0][0], selected[0][1]) : null;
```

Comment:
I'd use a bit readable API here. My suggestion:
```suggestion
    const selectedCell = this.hot.getSelectedRangeLast()?.highlight;
    const TD = selectedCell ? this.hot.getCell(selectedCell.row, selectedCell.col) : null;
```